src/main/java/com/based/basedsurvey/controller/WebEditSurveyController.java

```diff
@@ -78,6 +79,15 @@ public String enableButtons(@RequestParam long id) {
         var survey = ControllerHelperClass.getSurvey(id);
         survey.setStatus(Survey.SurveyStatuses.BEING_FILLED);
         surveyRepository.save(survey);
```

Comment:
I wonder if you should save the survey after all the for loop stuff you added (right before the return)